src/SDKs/Network/Network.Tests/Tests/ApplicationGatewayTests.cs

```diff
@@ -610,7 +610,7 @@ public void ApplicationGatewayApiTest()
                 Assert.NotEmpty(availableWAFRuleSets.Value[0].RuleGroups);
                 Assert.NotNull(availableWAFRuleSets.Value[0].RuleGroups[0].RuleGroupName);
                 Assert.NotEmpty(availableWAFRuleSets.Value[0].RuleGroups[0].Rules);
-                Assert.NotNull(availableWAFRuleSets.Value[0].RuleGroups[0].Rules[0].RuleId);
+                // Assert.NotNull(availableWAFRuleSets.Value[0].RuleGroups[0].Rules[0].RuleId);
```

Comment:
Why was this commented out?

Reply:
The `NotNull` is being applied on a literal type (which is never null).
This needs to be fixed by the RP